### PR TITLE
blog: Project Atomic Docker Hub Namespace Changes

### PR DESCRIPTION
**Pull request by ashcrow**
From atomic-wg discussion many weeks ago.

### CODE REVIEW

**Comment by jasonbrooks** [comment]
The first `it's` should have no apostrophe, but otherwise, LGTM

**Comment by ashcrow** [comment]
Updated

File: source/blog/2018-01-04-project-atomic-docker-hub-namespace-changes.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+READMORE
+
+## Pruning
+The first change to the namespace will be image pruning. Today there are 18 images that are available in the namespace. Images which are not being actively built and pulled will be removed from the Project Atomic namespace. By removing older and no longer used images from the registry it should be easier to follow what images are in development.
+
+## Automated Building
+New images, as well as images which are currently active, will start to be built upon source code changes. This means that images provided in the Project Atomic namespace will represent the master branch in source control. This change will allow users to pull the latest images to test and play with without the need to rebuild locally. It also means that the images available in the Project Atomic Docker Hub namespace will be **bleeding edge** and should be considered similar to rpm packages in Rawhide. If you are wanting to use a supported image you should use an official image.
```
Comment (dustymabe) [inline source/blog/2018-01-04-project-atomic-docker-hub-namespace-changes.md]: "official image". See section below about what an official image is.

**Comment by dustymabe** [comment]
/me still prefers to use 'devel' somewhere in the name of these containers. I have a few arguments for why:

- users won't obviously know they shouldn't use them long term
- maintainers have less incentive to get things into official registries
- there is more confusion when a user reports an issue what container they are using

These are arguments I have made before and have been overruled, so I'll be quiet now.

LGTM

**Pull request merged**